src/main/java/net/sf/jabref/importer/fileformat/OvidImporter.java

```diff
@@ -199,6 +199,15 @@ public boolean isRecognizedFormat(InputStream stream) throws IOException {
                     } else if (content.contains("Conference Paper")) {
                         h.put("entrytype", "inproceedings");
                     }
+                } else if (fieldName.indexOf("Language") == 0) {
```

Comment:
why not use startsWith? I think this is more readable as a combination of indexOf and index comparison to 0.